nginx-selfsigned.crt

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
could you try to generate all this certificates into the script? Yes - it's possible ;)

Reply:
I'll try to do that.